tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -53,6 +53,13 @@ var _ = Describe("odo devfile create command tests", func() {
 			componentNamespace := helper.RandString(6)
 			helper.CmdShouldPass("odo", "create", "java-openliberty", "--project", componentNamespace)
 		})
+
+		It("should fail tocreate the devfile component if --project value is 'default'", func() {
```

Comment:
@niconosenzo could you please give a single space in `tocreate`.

Reply:
@prietyc123 thanks, fixed.